packages/app-mobile/components/NoteList.tsx

```diff
@@ -91,7 +92,13 @@ class NoteListComponent extends Component<NoteListProps> {
 				keyExtractor={item => item.id}
 			/>;
 		} else {
-			if (!Folder.atLeastOneRealFolderExists(this.props.folders)) {
+			const noteSourceParentId = this.props.notesSource ? JSON.parse(this.props.notesSource).parentId : null;
+			const noteSourceFolder = this.props.folders.find(folder => folder.id === noteSourceParentId);
+
+			if (noteSourceParentId === getTrashFolderId() || itemIsInTrash(noteSourceFolder)) {
```

Comment:
After the recent fix to `itemIsInTrash`, the only thing that should be needed here is a call for `itemIsInTrash()`. Please confirm

Reply:
done, I have removed the extra check for trash folder